pyproject.toml

```diff
@@ -19,13 +20,21 @@ keywords = ['shipchain', 'transmission']
 [tool.poetry.dependencies]
 python = "^3.6"  # Compatible python versions must be declared here
 
+aws-requests-auth = "~0.4"
+boto3 = "~1.9"
+cryptography = "~2.5"
 django = "^2.2.4"
+django-enumfields = "==0.10.0"
```

Comment:
I have made an attempt to use very loose version pinning in this project, because we don't want to unnecessarily prevent projects that depend on this package from updating shared dependencies. We should only pin to major versions in most cases (when the dependency follows semver).